webapp/src/client.ts

```diff
@@ -155,17 +159,8 @@ export default class CallsClient extends EventEmitter {
 
         ws.on('join', async () => {
             console.log('join ack received, initializing connection');
-            let config;
-            try {
-                const resp = await axios.get(`${getPluginPath()}/config`);
-                config = resp.data;
-            } catch (err) {
-                console.log(err);
-                this.ws?.close();
-                return;
-            }
-
-            const iceServers = config.ICEServers?.length > 0 ? [{urls: config.ICEServers}] : [];
+            const configServers = getIceServers(this.getState());
+            const iceServers = configServers?.length > 0 ? [{urls: configServers}] : [];
```

Comment:
Is there a specific reason we want to be requesting fresh iceServers on `join`, or was that just a way to get the data? We can still request it on join if we want to.